src/auth/components/game-img/game-img.ts

```diff
@@ -0,0 +1,37 @@
+import Vue from 'vue';
+import Component from 'vue-class-component';
+import { State } from 'vuex-class';
+import AppGameThumbnailImg from '../../../_common/game/thumbnail-img/thumbnail-img.vue';
+import { Store } from '../../store/index';
+
+@Component({
+	name: 'GameCoverCredits',
+	components: {
+		AppGameThumbnailImg,
+	},
+})
+export default class GameCoverCredits extends Vue {
+	@State coverGame: Store['coverGame'];
+
+	get gameTitle() {
```

Comment:
just include these in the template within the interpolations directly, no need for getters here since they're simple access